cognite/neat/workflows/steps/lib/graph_loader.py

```diff
@@ -51,6 +61,75 @@ def run(self, rules: RulesData, cdf_client: CogniteClient) -> None:
         upload_labels(cdf_client, rules.rules, extra_labels=["non-historic", "historic"])
 
 
+class GenerateCDFNodesAndEdgesFromGraph(Step):
+    """
+    The step generates nodes and edges from the graph
+    """
+
+    description = "The step generates nodes and edges from the graph"
+    category = CATEGORY
+
+    configuration_templates: ClassVar[list[WorkflowConfigItem]] = [
+        WorkflowConfigItem(
+            name="nodes_and_edges_generation.graph_name",
+            value="source",
+            label=("The name of the graph to be used for matching." " Supported options : source, solution"),
+        ),
+    ]
+
+    def run(self, rules: RulesData, graph: SourceGraph | SolutionGraph) -> (FlowMessage, Nodes, Edges):
```

Comment:
I think you need to do this
```suggestion
    def run(self, rules: RulesData, graph: SourceGraph | SolutionGraph) -> tuple[FlowMessage, Nodes, Edges]:
```

Reply:
The definition of output is correct, 

If one adds `tuple` is incorrect here is how it is rendered in UI:
![Screenshot 2023-08-28 at 08 03 14](https://github.com/cognitedata/neat/assets/35523348/e9560ae9-89b3-4f3b-a875-1e684c3bc4b8)

Alt, if one use the existing definition of output (i.e. without `tuple`):

![Screenshot 2023-08-28 at 08 07 22](https://github.com/cognitedata/neat/assets/35523348/3e28c013-d411-4f5b-aec3-17bf85cea8b3)